packages/playground/kit-demo-app/src/routes/sverdle/+page.svelte

```diff
@@ -67,9 +67,11 @@
 		const key = /** @type {HTMLButtonElement} */ (event.target).getAttribute('data-key');
 
 		if (key === 'backspace') {
+			// eslint-disable-next-line svelte/no-reactive-reassign
```

Comment:
is this present in the current `sv create` template as well? if yes we should fix it there too. But i hope it uses code that doesn't require this and we can just recreate kit-demo-app here.

Reply:
The CLI repository uses Svelte5 Runes, allowing for more accurate checks, so ESLint errors shouldn’t occur.

Reply:
we should update kit-demo-app to svelte5 syntax here then too. but that can be done in a separate PR